<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventDialog</class>
 <widget class="QDialog" name="EventDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Event Editor</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>380</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAddEvent">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>240</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Event</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelTimeStart">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Time Start</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeEditTimeStart">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>75</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelTimeEnd">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Time End</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeEditTimeEnd">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>125</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelDayOfWeek">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Day of Week</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxDayOfWeek">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>175</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Sunday</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Monday</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Tuesday</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Wednesday</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Thursday</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Friday</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Saturday</string>
    </property>
   </item>
  </widget>
  <widget class="QTextEdit" name="textEditEventDetails">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>80</y>
     <width>221</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonRemoveEvent">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>310</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove Event</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelStudent">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Student:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelStudentName">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Student Name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelDescription">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>40</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Description</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidgetStudentEventList">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>341</width>
     <height>161</height>
    </rect>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="currentRow">
    <number>-1</number>
   </property>
   <property name="sortingEnabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EventDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EventDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
